packages/lesswrong/components/tagging/TagPageUtils.tsx

```diff
@@ -19,7 +20,10 @@ export const tagPageHeaderStyles = (theme: ThemeType) => ({
 
 export const tagPostTerms = (tag: Pick<TagBasicInfo, "_id" | "name"> | null, query: any) => {
   if (!tag) return
+  
+  // EA Forum defaults to top sort order for tag pages
   return ({
+    ...(isEAForum && {sortedBy: "top"}),
```

Comment:
This should probably be `isFriendlyUI`, as that's what's used to decide which tag page to show